Ical.Net/Evaluation/EvaluationOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿//
+// Copyright ical.net project maintainers and contributors.
+// Licensed under the MIT license.
+//
+
+namespace Ical.Net.Evaluation;
+public class EvaluationOptions
+{
+    /// <summary>
+    /// The maximum number of increments to evaluate without finding a recurrence before
+    /// evaluation is stopped exceptionally. If null, the evaluation will continue indefinitely.
+    /// </summary>
+    /// <remarks>
+    /// This option only applies to the evaluation of RecurrencePatterns.
+    ///
+    /// If the specified number of increments is exceeded without finding a recurrence, an
+    /// exception of type <see cref="Ical.Net.Evaluation.MaxIncrementsExceededEvaluationException"/> will be thrown.
+    /// </remarks>
+    public int? MaxIncrementCount { get; set; }
```

Comment:
As it's not a counter, would sth. like `NoPatternMatchLimit` be more intuitive?
Likewise sth. like `NoPatternMatchLimitExceededException`?

Reply:
Certainly better!

Reply:
Your new naming is the best